code/controllers/subsystem/research.dm

```diff
@@ -351,22 +357,30 @@ SUBSYSTEM_DEF(research)
 				result[i] *= eff
 				bitcoins[i] = bitcoins[i]? bitcoins[i] + result[i] : result[i]
 	else
-		for(var/obj/machinery/rnd/server/miner in servers)
+		for(var/obj/machinery/rnd/server/miner in BOSservers)
+			if(miner.working)
+				BOSbitcoins = BOSsingle_server_income.Copy()
+				//break
```

Comment:
```suggestion
				break
```
Why was this break removed? It literally just wastes processing time to remove it.

Reply:
will take a look at the rest.

Reply:
Ok honestly no idea what I'm doing apart from removing the breaks. Tried for a bit and failed. Not a CS student/worker here as you can see so you'll have to explain this in more detail if you want me to actually manage to do anything.

Reply:
Oh shit the CI finished?  *slaps merge without looking*  Yeeeeeeeeeeeeeeeeeeeeeeeeee haawwwwwwww